docs/50_hub.md

```diff
@@ -0,0 +1,17 @@
+# Using the DANDI Hub
+
+[DANDI Hub](http://hub.dandiarchive.org) is a JupyterHub instance in the cloud to interact with the data stored in DANDI, and is free to use for exploratory analysis of data on DANDI. Note that DANDI Hub is not intended for significant computation, but provides a place to introspect Dandisets and to perform some analysis and visualization of data.
+
+## Registration
+
+To use the [DANDI Hub](http://hub.dandiarchive.org), you must first register for an account using the [DANDI website](http://dandiarchive.org).
+
+## Choosing a server option
+
+When you start up the DANDI Hub, you will be asked to select across a number of server options. For basic exploration, Tiny or Base would most likely be appropriate. The DANDI Hub also currently offers Medium and Large options, which have more available memory and compute power. The "T4 GPU inference" server comes with an associated T4 GPU, and is intended to be used for applications that require GPU for inference. We request that users of this server be considerate of their usage of the DANDI Hub as a free community resource. Training large deep neural networks is not appropriate. A "Base (MATLAB)" server is also available, which provides a MATLAB cloud installation.
+
+## Example notebooks
+
+The best way to share analyses on DANDI data is through the DANDI example notebooks. These notebooks are organized by `<DANDI id>/<group>/<analysis>/<notebook.ipynb>`. Dandiset contributors are encouraged to use these notebooks to demonstrate how to read, analyze, and visualize the data, and how to produce figures from associated scientific publications.
```

Comment:
Split to one sentence per line and point to example-notebooks repo

```suggestion
The best way to share analyses on DANDI data is through the DANDI example notebooks.
These notebooks are maintained in https://github.com/dandi/example-notebooks repository which provides more information about their organization.
Dandiset contributors are encouraged to use these notebooks to demonstrate how to read, analyze, and visualize the data, and how to produce figures from associated scientific publications.
```

Reply:
@yarikoptic is there a reason you removed the sentence: "These notebooks are organized by `<DANDI id>/<group>/<analysis>/<notebook.ipynb>`."

Reply:
I geuss you are right, this is no longer needed since this information is now in the [example notebooks readme](https://github.com/dandi/example-notebooks)